src/lib.rs

```diff
@@ -3,6 +3,7 @@
 extern crate bitcoin;
 extern crate bitcoind;
 
+pub mod default_config_params;
```

Comment:
I feel its better not to have a separate module for this. Its redundant. These functions should be in the `src/util.rs` module. 

Tests should be written in each individual modules,  by calling the util functions. Ex:

- Default wallet file load/write in `wallet.rs` module.
- Default Taker Config load/write in `taker/config.rs` module.
And so on.